app/frontend/src/components/admin/AccessGrant.vue

```diff
@@ -45,6 +45,38 @@
                 outlined
               />
             </v-col>
+            <v-col sm="6" lg="4" class="pt-2">
+              <label>Ministry</label>
+              <v-text-field
+                v-model="ministry"
+                dense
+                flat
+                hint="Acronym of the ministry this new Acronym is under"
```

Comment:
minor point.. maybe this should read: 'Ministry for client application (eg: NRIDS)'

Reply:
That or "Ministry this application is under"